typescript/infra/src/agents/aws/key.ts

```diff
@@ -231,6 +231,8 @@ export class AgentAwsKey extends CloudAgentKey {
     if (!keyId) {
       throw Error('Key ID not defined');
     }
+    // @ts-ignore provider.getFeeData type mismatch due to
+    // aws-kms-ethers-signer using ethers 5.6 and this package using ethers 5.7
```

Comment:
🤔
should we use npm overrides?
https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides

Reply:
I tried and couldn't get it to work :(